datalad/crawler/nodes/annex.py

```diff
@@ -336,7 +336,7 @@ def __init__(self, path=None,
         self._providers = Providers.from_config_files()
         self.yield_non_updated = yield_non_updated
 
-        if (not allow_dirty) and self.repo.dirty:
+        if (not allow_dirty) and self.repo.dirty():
```

Comment:
And: If you want that property, be aware you save just 5 Bytes by `dirty` instead of `is_dirty()`. You need to make sure, that you use it sufficiently often to compensate for the entire property definition! :-D